src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -189,14 +189,21 @@ private void UpdateServerType()
         private void ApplyViewFilter(Predicate<object> filter) =>
             IssuesView.Filter = filter;
 
-        private bool ActiveDocumentFilter(object viewModel)
+        private bool NotSuppressedIssuesInCurrentDocumentFilter(object viewModel)
         {
             if (string.IsNullOrEmpty(activeDocumentFilePath))
             {
                 return false;
             }
 
-            var allFilePaths = ((ITaintIssueViewModel) viewModel).TaintIssueViz.GetAllLocations()
+            var issueViz = ((ITaintIssueViewModel) viewModel).TaintIssueViz;
+
+            if (issueViz.IsSuppressed)
+            {
+                return false;
+            }
```

Comment:
This current change is not reactive; if SSE changes it, the list won't auto-refresh until document changed event is raised. I will fix it in the next PR.